front-end/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@cloudscape-design/components": "^3.0.388",
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
```

Comment:
These should be in devDependencies. Read up about it and reply to this comment why they should be in dev dependencies.

Reply:
devDependencies are modules that are required during runtime. In this case, the modules jest-dom basically the testing modules are required during development. Hence they should be in devDependencies.